orca-deploymentmonitor/src/main/java/com/netflix/spinnaker/config/DeploymentMonitorDefinition.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.config;
+
+import com.netflix.spinnaker.orca.deploymentmonitor.DeploymentMonitorService;
+import lombok.Data;
+
+@Data
+public class DeploymentMonitorDefinition {
+  /** Unique ID of this deployment monitor */
+  private String id;
+
+  /** Human friendly name of this deployment monitor */
+  private String name;
+
+  /** Base URL for this deployment monitor */
+  private String baseUrl;
+
+  /**
+   * Maximum number of minutes this deployment monitor is allowed to respond to the /evaluateHealth
+   * request. Failure to respond in this time frame will result in deployment failure.
+   */
+  private int maxAnalysisMinutes = 30;
```

Comment:
Prefer the use of `Duration` for any time unit properties.

Reply:
@robzienert does this mean you prefer it to be ISO duration format or what would be the unit? Minutes seemed pretty intuitive to me, but if there is a patter we are going to/striving for, am all for that!

Reply:
@robzienert means using a Duration object instead of an int variable:

`Duration maxAnalysis = Duration.ofMinutes(30)`